values-schema.yaml

```diff
@@ -2409,12 +2427,71 @@ properties:
                     $ref: '#/definitions/google/definitions/accountJson'
                     description: A service account key in json format for managing a DNS zone.
                   project:
-                    $ref: '#/definitions/google/definitions/projectId'
-                required:
-                  - serviceAccountKey
-                  - project
+                    $ref: '#/definitions/google/definitions/project'
+                required: [project, serviceAccountKey]
             required:
               - google
+          - title: Digital Ocean
+            properties:
+              digitalocean:
+                properties:
+                  apiToken:
+                    type: string
+                    x-secret: true
+                required:
+                  - apiToken
+            required:
+              - digitalocean
+          - title: CloudFlare
+            properties:
+              cloudflare:
+                properties:
+                  apiToken:
+                    type: string
+                    x-secret: true
+                  apiSecret:
+                    type: string
+                    x-secret: true
+                    description: Required when Email is set.
+                  email:
+                    $ref: '#/definitions/email'
+                    description: Required when Email is set.
+                  proxied:
+                    type: boolean
+                    x-default: true
+                oneOf:
+                  - required: [apiToken]
+                  - required: [apiSecret, email]
+                  - not:
+                      anyOf:
+                        - required: [apiToken]
+                        - required: [apiSecret]
+                        - required: [email]
+            required:
+              - cloudflare
+          - title: Other
+            properties:
+              other:
```

Comment:
Could you provide some examples for other DNS, so we can test it

Reply:
already tested. But I found one major issue.
The settings for the `other` provider shall be stored in `secrets.settings.yaml` but they are stored in `settings.yaml`
At the beginning I thought that adding `x-secret` would solve the issue but it does not support `objects`, so we need to figure it out how to store DNS configuration securel.